lib/core/src/lib/assets/silent-refresh.html

```diff
@@ -2,5 +2,8 @@
     <title>Alfresco silent static page</title>
 </head>
 <body>
+    <script>
```

Comment:
Seems quite workaround-y, but for the sake of release every solution is good solution! 😅 
Please let's create a ticket for post-release to investigate how this could be cleaned up.

Reply:
I've got that solution from the [official documentation ](https://manfredsteyer.github.io/angular-oauth2-oidc/docs/additional-documentation/silent-refresh.html)

Reply:
we were doing kind of the same the only difference was that js api was injecting the post into a div dynamically on silentRefresh page. The only concerne is... have we tested it? have u checked it the refreshToken is present or the idToken got refreshed ?

Reply:
I tested yesterady locally with studio app, with this change it refresh the token before expire and all the other API call work good, of course I should try it merging the upstream